### PR TITLE
Add a "configuration" suite to perform the preliminary setup

### PR DESCRIPTION
**Pull request by fedepaol**
Also, add a test-run.sh entry point that calls both suites and forward them the parameters.
Use the test-run.sh as part of make functests.

Catch the reporter error instead of panicking.

### CODE REVIEW

**Comment by slintes** [comment]
/lgtm

giving others the chance to review as well
/hold

File: configsuite/test_suite_test.go [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+	"log"
+	"os"
+	"path"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+
+	_ "github.com/openshift-kni/performance-addon-operators/functests/utils/namespaces"
```
Comment (slintes) [inline configsuite/test_suite_test.go:16]: why is this needed?

Comment (fedepaol) [reply configsuite/test_suite_test.go]: It's a leftover, removed it.

**Comment by slintes** [comment]
nice :+1: 

/lgtm
